eclass/cargo.eclass

```diff
@@ -179,6 +179,7 @@ cargo_src_install() {
 		--root="${ED}/usr" $(usex debug --debug "") "$@" \
 		|| die "cargo install failed"
 	rm -f "${ED}/usr/.crates.toml"
+	rm -f "${ED}/usr/.crates2.json"
```

Comment:
Interesting, haven't encountered packages doing that yet, I guess it will come with next rust/cargo versions and we can future-proof eclass.